dev-docs/bidders/fidelity.md

```diff
@@ -8,7 +8,6 @@ biddercode: fidelity
 media_types: banner
 gdpr_supported: true
 usp_supported: true
-gvl_id: 408
```

Comment:
Why are you removing the GVLID from the fidelity adapter? 408 seems valid for them -- https://vendorlist.consensu.org/v2/vendor-list.json

```
408: {
id: 408,
name: "Fidelity Media",
purposes: [
1,
2,
7
],
legIntPurposes: [ ],
flexiblePurposes: [
2,
7
],
specialPurposes: [ ],
features: [ ],
specialFeatures: [ ],
policyUrl: "https://fidelity-media.com/privacy-policy/",
deletedDate: "2020-08-25T00:00:00Z",
cookieMaxAgeSeconds: -100,
usesNonCookieAccess: false
},
```

Reply:
The fidelity adapter still refers to the id

const FIDELITY_VENDOR_ID = 408;

Reply:
@bretg Thanks, I'll look into and fix it on Monday

Reply:
@bretg  it is fixed now. Please, re-review.